bluesky_kafka/kafka.py

```diff
@@ -70,8 +115,18 @@ class RemoteDispatcher(Dispatcher):
 
     Parameters
     ----------
-    address : str or tuple
-        Address of a Kafka server as a string like ``'127.0.0.1:9092'``
+    topics: list
+        List of topics as strings such as ["topic-1", "topic-2"]
+    bootstrap_servers : str
```

Comment:
I have a mild preference for leaving address first, since it's required anyway, for easy of transition between 0MQ and Kafka.